test/bin/scenario.sh

```diff
@@ -214,8 +215,7 @@ run_tests() {
     echo "Running tests with $# args" "$@"
 
     if [ ! -d "${RF_VENV}" ]; then
-        error "RF_VENV (${RF_VENV}) does not exist, create it with: ${ROOTDIR}/scripts/fetch_tools.sh robotframework"
-        exit 1
+        "${ROOTDIR}/scripts/fetch_tools.sh" robotframework
```

Comment:
We can't do this in the scenario script because in CI several copies of this will eventually run in parallel. We could do it explicitly in the phase script instead.